compiler/luci/pass/src/RemoveUnnecessaryTransposeNetPass.cpp

```diff
@@ -268,9 +293,8 @@ bool TaggedShapeAnalyzer::verify_tag() const
  * @brief Initialize the class members and check under conditions
  *
  *  Condtiions that have to be met for analyzer
- *    c1: input rank >= output rank
  *    c2: The 'perm' of tranpose should be a CircleConst* type
- *    c3: The input shape and the reshape node's shape should be known
+ *    c3: All extracted shapes (named as '*_shape_v' in member variable) should be known
```

Comment:
(note) diff in `init()`
* `c1` is released.
* `c3` is a bit modified, because there are new member variables.

Reply:
May need to update numbering.

Reply:
I'll update numbering in next PR.